src/com/cft/contactmerge/MergeFileExporter.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * MergeFileExporter will serialize the list of matched contacts into the file
+ * type requested using the column map to identify where each property is stored
+ * in the file.
+ */
```

Comment:
I would use the following descriptiong:
MergeFileExporter will be responsible for taking the final matching results of both imported files, and creating an exported file of the merged records.

Reply:
Updated the text.